README.md

```diff
@@ -1 +1,204 @@
-# OpenSLO
\ No newline at end of file
+# OpenSLO
+
+## Table of Contents
+
+- [OpenSLO](#openslo)
+  - [Introduction](#introduction)
+  - [Specification](#specification)
+    - [Goals](#goals)
+    - [Object Types](#object-types)
+      - [General Schema](#general-schema)
+        - [Notes](#notes)
+      - [SLO](#slo)
+        - [Notes](#notes-1)
+        - [Objectives](#objectives)
+      - [Service](#service)
+
+## Introduction
+
+The intent of this document is to outline the OpenSLO specification.
+
+The goal of this project is to provide an open specification for defining and
+interfacing with SLOs to allow for a common, giving a set vendor-agnostic
```

Comment:
s/for a common,/for a common approach,/
s/giving a set/and giving a/

Reply:
fixed